src/libs/actions/PersonalDetails.ts

```diff
@@ -382,7 +382,14 @@ function updateAvatar(file: File | CustomRNImageManipulatorResult) {
         },
     ];
 
-    const parameters: UpdateUserAvatarParams = {file};
+    let parameters: UpdateUserAvatarParams = {file};
+
+    // We need to remove the base64 value from the file object, as it is causing crashes on Release builds.
+    // More info: https://github.com/Expensify/App/issues/37963#issuecomment-1989260033
+    if ('base64' in file) {
```

Comment:
Is this the only place we need to fix this? Should we be putting this somewhere in the `cropOrRotateImage` or a util?

Reply:
Right, putting it in the `cropOrRotateImage` will prevent similar issues in the future. I've moved it.